controller.go

```diff
@@ -221,6 +224,12 @@ func loadSettings() error {
 		Default(aws.DefaultHealthCheckInterval.String()).DurationVar(&healthCheckInterval)
 	kingpin.Flag("health-check-timeout", "sets the health check timeout for the created target groups. The flag accepts a value acceptable to time.ParseDuration").
 		Default(aws.DefaultHealthCheckTimeout.String()).DurationVar(&healthCheckTimeout)
+	kingpin.Flag("alb-healthy-threshold-count", "The number of consecutive successful health checks required before considering an unhealthy target healthy. The range is 2–10. The default is 5. (ALB only)").
+		Default(strconv.FormatUint(aws.DefaultAlbHealthyThresholdCount, 10)).UintVar(&albHealthyThresholdCount)
+	kingpin.Flag("alb-unhealthy-threshold-count", "The number of consecutive failed health checks required before considering a target unhealthy. The range is 2–10. The default is 2. (ALB only)").
+		Default(strconv.FormatUint(aws.DefaultAlbUnhealthyThresholdCount, 10)).UintVar(&albUnhealthyThresholdCount)
+	kingpin.Flag("nlb-healthy-threshold-count", "The number of consecutive successful or failed health checks required before considering an target healthy or unhealthy. The range is 2 to 10. The default is 3. (NLB only)").
+		Default(strconv.FormatUint(aws.DefaultNlbHealthyThresholdCount, 10)).UintVar(&nlbHealthyThresholdCount)
```

Comment:
I think the default value is rendered in the help message:
```
  --alb-healthy-threshold-count=5  
                                 The number of consecutive successful health
                                 checks required before considering an unhealthy
                                 target healthy. The range is 2–10. The default
                                 is 5. (ALB only)
  --alb-unhealthy-threshold-count=2  
                                 The number of consecutive failed health checks
                                 required before considering a target unhealthy.
                                 The range is 2–10. The default is 2. (ALB only)
  --nlb-healthy-threshold-count=3  
                                 The number of consecutive successful or failed
                                 health checks required before considering an
                                 target healthy or unhealthy. The range is 2 to
                                 10. The default is 3. (NLB only)
```
so lets remove `The default is ...`.

For `--nlb-healthy-threshold-count` it should be `considering _a_ target` and I think we should use the same format for the range as for ALB: `The range is 2–10` for consistency (I get that it is different in the AWS docs).

Reply:
Done, thanks for the review

Reply:
Thank you. `_an_ target` is still there though.

Reply:
thanks, sorry